### ISSUE TITLE
Stuck at the last step of makedb

### ISSUE THREAD

**Issue by AlexSongh** [bioinformatics-centre/kaiju#234]
Hi I have a question about the step of Creating the reference database and index

Here is my output log file 
<img width="650" alt="image" src="https://user-images.githubusercontent.com/55120416/177833754-1befdfc9-d878-4603-bcb6-c22dac24e005.png">
I used 5 cores, RAM of 800G and a walltime of 30 hours and it still didn't finish and stuck at this step. 

Do you have any recommendations to solve this? Thanks!

**Comment by pmenzel** [comment]
Is it automatically terminated by the cluster when the requested wall clock time runs out?

I had this the last time I made the nr index: `Elapsed (wall clock) time (h:mm:ss or m:ss): 61:01:03`

Your mileage will vary depending on download speed and CPU speed. Maybe set it to 70 hours or more and try again.
When you already have downloaded nr.gz file, you can add option `--no-download` to makedb to reuse the existing file.

**Comment by ipetrushin** [comment]
I totally agree with this issue. The process `wget -c -nv --show-progress -P nr_euk...` hanging several hours, while file was finished (I even checked the md5 sum, it was correct). Re-runnig with `--no-download` says `Missing file prot.accession2taxid.gz`
